.babelrc

```diff
@@ -8,11 +8,6 @@
     "@babel/preset-flow"
   ],
   "plugins": [
-    [
-      "@babel/plugin-transform-runtime", {
```

Comment:
You dropped the usage of babel runtime, but I think you still left it on the package.json